manifests/pear/module.pp

```diff
@@ -59,13 +59,13 @@
   }
 
   $pear_exec_unless = $ensure ? {
-    present => "pear info ${pear_source}",
+    present => "pear shell-test ${pear_source} > 0",
     absent  => undef
   }
 
   $pear_exec_onlyif = $ensure ? {
     present => undef,
-    absent  => "pear info ${pear_source}",
+    absent  => "pear shell-test ${pear_source} > 0",
```

Comment:
Oops, that's an excellent question. There's supposed to be a literal '>' in that command line, not a redirect like there is currently.